zio-http/src/main/scala/zio/http/api/internal/RichTextCodec.scala

```diff
@@ -186,6 +187,33 @@ object RichTextCodec {
    */
   val empty: RichTextCodec[Unit] = Empty
 
+  val comma: RichTextCodec[Char] = char(',')
+
+  val dot: RichTextCodec[Char] = char('.')
+
+  val slash: RichTextCodec[Char] = char('/')
+
+  val star: RichTextCodec[Char] = char('*')
+
+  val colon: RichTextCodec[Char] = char(':')
+
+  val semicolon: RichTextCodec[Char] = char(';')
+
+  def commaSeparatedMultiValues[A](codec: RichTextCodec[A]): RichTextCodec[Chunk[A]] =
```

Comment:
This is called `rep1sep` in some libraries and would be a method on `RichTextCodec`, with it accepting the separator character (as a separate `RichTextCodec[Unit]`).

Probably worth doing it that way so the code will be more familiar.

Reply:
Also, generally a good idea to save companion object for constructors rather than unary operators like this one (which properly belong on the data type, for easy dot-syntax).